src/utils/index.ts

```diff
@@ -24,6 +24,19 @@ const CURRENCY_MAP = new Map([
   ['AUD', { icon: 'aud', name: 'Australian Dollar' }],
 ]);
 
+export const domains = [
+  'deriv.com',
```

Comment:
Do you need to include deriv.be and deriv.me as well?

Reply:
added